docs/contributing/02-maintainers/050-operations.md

```diff
@@ -125,7 +125,7 @@ Any maintainer should be empowered to take ownership of an incident if they wish
 - (always, immediately) send a Slack message to #alert
 - (always, immediately) notify the primary responder
 - (if not acknowledged within 10 minutes) notify the secondary responder
-- (if not acknowledged) Repeat everything above except for the slack message up to 6 times.
+- (if not acknowledged) Repeat everything above except for the Slack message up to 6 times.
```

Comment:
Discord?

Reply:
no, it is about the Alert process. it should still be in Slack (except the part of letting everyone know on #dev)